DataGrid: update pager.visible property help topic (T1165507) (#5034)
DevExpress/devextreme-documentation: api-reference/10 UI Components/GridBase/1 Configuration/pager/visible.md

Specifies whether the pager is visible.

---

> The pager is only visible if paging is [enabled](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/paging/enabled.md '{basewidgetpath}/Configuration/paging/#enabled') and the total row count is greater than the [pageSize](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/paging/pageSize.md '{basewidgetpath}/Configuration/paging/#pageSize') property. If you choose a sufficient large page size (for example "All" supported in [allowedPageSizes](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#allowedPageSizes)) and the data doesn't increase (especially with state storing enabled), the pager will be hidden permanently.

Reviewer: ```suggestion
The pager is only visible if paging is [enabled](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/paging/enabled.md '{basewidgetpath}/Configuration/paging/#enabled') and the total row count is greater than the [pageSize](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/paging/pageSize.md '{basewidgetpath}/Configuration/paging/#pageSize') property. If you choose a large page size (for example "All" supported in [allowedPageSizes](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#allowedPageSizes)) and the data does not increase (for example, with state storing enabled), the pager is hidden permanently.
```
